src/main/java/org/lwjglx/opengl/Display.java

```diff
@@ -169,7 +169,9 @@ public void invoke(long window, int key, int scancode, int action, int mods) {
                 cancelNextChar = false;
                 if (key > GLFW_KEY_SPACE && key <= GLFW_KEY_GRAVE_ACCENT) { // Handle keys have a char. Exclude space to
                                                                             // avoid extra input when switching IME
-                    if ((GLFW_MOD_CONTROL & mods) != 0 && (GLFW_MOD_ALT & mods) == 0) { // Handle ctrl + x/c/v.
+                    if ((GLFW_MOD_SUPER & mods) != 0) {
+                        Keyboard.addGlfwKeyEvent(window, key, scancode, action, mods, (char) key);
```

Comment:
`cancelNextChar = true'` is probably also needed here because you already specify a char code.

Reply:
At least on macOS, the Cmd+<key> never sends the char event (before or after the fix), which is the cause of the bug.

Reply:
I will have to test this on other platforms to make sure this code doesn't break windows/linux

Reply:
Probably just linux, as `GLFW_MOD_SUPER` is mapped to META, aka Cmd on macOS and Win key on Windows, which can hardly be in combination with other keys.